src/features/checker/components/ProjectReviewList/ProjectReviewList.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconLabel } from "@/components/IconLabel";
+import { Button } from "@/primitives/Button";
+import { CircleStat } from "@/primitives/Indicators";
+import { ListGrid, ListGridColumn } from "@/primitives/ListGrid";
+
+import { getReviewsCount } from "../../utils/getReviewsCount";
+import { ProjectReview } from "./types";
+
+export interface ProjectReviewListProps {
+  reviewer: `0x${string}`;
+  projects: ProjectReview[];
+}
+
+export const ProjectReviewList = ({ reviewer, projects }: ProjectReviewListProps) => {
+  const columns: ListGridColumn<ProjectReview>[] = [
+    {
+      header: "Project",
+      key: "project",
+      width: "2fr",
+      render: (item) => (
+        <div className="flex items-center gap-4">
+          <img
+            src={item.avatarUrl}
+            alt={item.name}
+            className="aspect-square size-12 rounded-sm"
+            onError={(event: React.SyntheticEvent<HTMLImageElement, Event>) => {
+              event.currentTarget.src = "/images/grey-image.png";
```

Comment:
can we use the skeleton from shadcn as the whole <img> instead of the gray-image?

Reply:
yes we can, I am gonna ask Melissa what to show there